features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -419,42 +419,44 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
     const PRIVATE_KEY_JWT: string = "private_key_jwt";
     const TLS_CLIENT_AUTH: string = "tls_client_auth";
 
-    useEffect(() => {
-        if (sharedOrganizationsList) {
-            return;
-        }
+    if (orgType !== OrganizationType.SUBORGANIZATION) {
```

Comment:
It's forbidden to use hooks inside the conditionals.

https://react.dev/warnings/invalid-hook-call-warning

let's move this conditional logic inside the hook instead.

eg: 

```
useEffect(() => {
  if ( orgType !== "suborg" ) {
    // logic
  }
}, [orgType]) // add orgType as a dependency
```

Reply:
Modified